keps/sig-scheduling/1819-scheduler-extender/README.md

```diff
@@ -233,6 +236,17 @@ type ExtenderFilterResult struct {
 
 The "filter" call may prune the set of nodes based on its filter plugins.
 
+Nodes in both `FailedNodesMap` and `FailedAndUnresolvableNodes` are
+unschedulable, except the nodes in the latter will be skipped in preemption
+phase.
+
+When multiple extenders are configured, unschedulable nodes will not be passed
+to subsequent extenders. This is like the scheduler framework running filter
+plugins, which stops on the first filter that reports unschedulable status.
+Users are encouraged to carefully order extenders to have the extender which
```

Comment:
It's recommended to order the extenders which may report `UnschedulableAndUnresolvable` ahead of others. This can improve the preemption efficiency.